r/rsparkling/tests/testthat/test_transforms.R

```diff
@@ -3,98 +3,89 @@ context("Test transformations of H2O frames and Spark frames in rsparkling")
 
 test_that("Test transformation from h2o frame to data frame", {
   sc <- spark_connect(master = "local[*]")
-  df = copy_to(sc,as.data.frame(t(c(1,2,3,4,"A"))))
-  df_hex = as_h2o_frame(sc,df)
-  df_tbl = as_spark_dataframe(sc,df_hex)
-
-  expect_equal(as.data.frame(tally(df_tbl))$n,nrow(df_hex))
-  expect_equal(ncol(df_tbl),ncol(df_hex))
-  expect_equal(colnames(df_tbl),colnames(df_hex))
-
-
+  df <- as.data.frame(t(c(1,2,3,4,"A"))) # workaround for sparklyr#316
+  sdf <- copy_to(sc, df)
+  hf <- as_h2o_frame(sc, sdf, strict_version_check=FALSE)
```

Comment:
`strict_version_check=FALSE` required to run tests because user h2o version might be different that h2o fixed in SW, other changes in this files are more cosmetic.